sources/engine/Stride.Graphics/GeometricPrimitives/GeometricPrimitive.Capsule.cs

```diff
@@ -161,7 +163,8 @@ public static GeometricMeshData<VertexPositionNormalTexture> New(float length =
 
                         var normal = new Vector3(dx, dy, dz);
                         var textureCoordinate = new Vector2(u * uScale, v * vScale);
-                        var position = radius * normal + new Vector3(0, deltaY, 0);
+                        var position = radius * normal;
+                        position.Y += deltaY;
```

Comment:
Why making this change? It is equivalent. I'd rather keep the file history clean and avoid changes that don't bring anything of value.